### PR TITLE
tld: add .tldcache command

### PR DESCRIPTION
**Pull request by dgw**
### Description
Allows viewing last update time for both caches, forcing the cached data to update, and clearing the cached data from both bot memory and the bot's database.

Perhaps a *bit* messier than it could be, but I'm tired of resetting my test bot's database values for tonight.

Not mentioned in commit log: This change means anyone running from source will have to manually delete the 'tld' plugin's stored plugin values from the database, as the timestamp format has changed.

### Checklist
- [x] I have read [CONTRIBUTING.md](https://github.com/sopel-irc/sopel/blob/master/CONTRIBUTING.md)
- [x] I can and do license this contribution under the EFLv2
- [x] No issues are reported by `make qa` (runs `make quality` and `make test`)
- [x] I have tested the functionality of the things this change touches

### CODE REVIEW

**Comment by dgw** [comment]
Yes, #1948 being accepted _would_ mean that I should change the imports in here.

**Comment by dgw** [comment]
That import thing that both LGTM and Travis flagged is exactly why I also opened #1948. Ignoring until that's either merged or dismissed with cause.

**Review by Exirel — APPROVED** [review]

**Comment by dgw** [comment]
I probably won't see until tonight, but that force-push _should_ pass CI now that #1948 got merged.

**Comment by dgw** [comment]
Looks like I already have something to fix when I get home: py2.7 doesn't have `datetime.timezone`. Fun.

File: sopel/modules/tld.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 """
 from __future__ import absolute_import, division, print_function, unicode_literals
 
-from datetime import datetime
+from datetime import datetime, timezone
```
Comment (Exirel) [inline sopel/modules/tld.py:12]:
```suggestion
from datetime import datetime
```

Comment (Exirel) [reply sopel/modules/tld.py:12]: HA I can't do the other suggestion, which is to do that:

```
import pytz
```

Right before the `import requests` down below.

File: sopel/modules/tld.py [hunk 2]
```diff
@@ -204,13 +205,13 @@ def _update_tld_data(bot, which):
         LOGGER.error("Asked to update unknown cache type '%s'.", which)
         return
 
-    now = datetime.now()
+    now = datetime.now(tz=timezone.utc)
```
Comment (Exirel) [inline sopel/modules/tld.py:208]:
```suggestion
    now = datetime.now(tz=pytz.utc)
```

**Review by Exirel — COMMENTED** [review]
Apart from that missing `import pytz` that should do it.

File: sopel/modules/tld.py (hunk 1 above)
Comment (dgw) [reply sopel/modules/tld.py]: Gotta love GitHub's artificial restrictions on PR reviews. 🙄

**Comment by dgw** [comment]
@Exirel Believe it or not, I had already come up with the exact same changes on my own by the time your suggestions hit my email inbox. (And even if not, I'm also too lazy to apply suggestions and rebase when I could just `git commit --amend --no-edit` 😏)

**Review by Exirel — APPROVED** [review]

**Pull request merged**